airflow/configuration.py

```diff
@@ -140,6 +140,8 @@ class AirflowConfigParser(ConfigParser):
         ('atlas', 'password'),
         ('smtp', 'smtp_password'),
         ('webserver', 'secret_key'),
+        # The following options are deprecated
+        ('core', 'sql_alchemy_conn')
```

Comment:
```suggestion
        ('core', 'sql_alchemy_conn'),
```
To fix static checks